pkgs/by-name/an/angular-language-server-bin/package.nix

```diff
@@ -0,0 +1,83 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchurl,
+  nodejs,
+  makeBinaryWrapper,
+  runCommand,
+  angular-language-server-bin,
+  writeShellApplication,
+  curl,
+  common-updater-scripts,
+  jq,
+  unzip,
+}:
+
+let
+  owner = "angular";
+  repo = "vscode-ng-language-service";
+in
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "angular-language-server-bin";
+  version = "18.2.0";
+  src = fetchurl {
+    name = "${finalAttrs.pname}-${finalAttrs.version}.zip";
+    url = "https://github.com/${owner}/${repo}/releases/download/v${finalAttrs.version}/ng-template.vsix";
+    hash = "sha256-rl04nqSSBMjZfPW8Y+UtFLFLDFd5FSxJs3S937mhDWE=";
+  };
+
+  nativeBuildInputs = [
+    unzip
+    makeBinaryWrapper
+  ];
+
+  buildInputs = [ nodejs ];
+
+  installPhase = ''
+    runHook preInstall
+    install -Dm755 server/bin/ngserver $out/bin/ngserver
+    install -Dm755 server/index.js $out/index.js
```

Comment:
Can we place those not at /?

Reply:
@SuperSandro2000 No because the upstream unwrapped `ngserver.bin` calls the index.js file with the relative path `..`. What would be the advantage of having them both at $out?

Reply:
I meant can we place them somewhere else like $out/share/angular-langugage-server so that don't pollute $out

Reply:
Alright, I put them under $out/lib.